### PR TITLE
[BEAM-46] Bigtable write streaming

### PR DESCRIPTION
**Pull request by ianzhou1**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

### CODE REVIEW

**Comment by ianzhou1** [comment]
R: @dhalperi @swegner 
Not sure what the JIRA issue number is... also wasn't sure what to add for populateDisplayData other than the Table ID.

**Comment by swegner** [comment]
This looks like your JIRA: [[BEAM-46] Unbounded sink for Google Cloud Bigtable](https://issues.apache.org/jira/browse/BEAM-46)

**Comment by swegner** [comment]
Display data implementation looks good. Would you mind adding an integration test in `runners/google-cloud-dataflow-java`, similar to [`DataflowBigQueryIOTest#testSinkPrimitiveDisplayData()`](https://github.com/apache/incubator-beam/blob/master/runners/google-cloud-dataflow-java/src/test/java/org/apache/beam/runners/dataflow/io/DataflowBigQueryIOTest.java#L79)?

File: sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java [hunk 1]
```diff
@@ -844,101 +843,50 @@ public final synchronized BigtableSource splitAtFraction(double fraction) {
...
+      logger.info("Wrote {} records", recordsWritten);
     }
 
     @Override
-    public Coder<Long> getWriterResultCoder() {
-      return VarLongCoder.of();
+    public void populateDisplayData(DisplayData.Builder builder) {
+      super.populateDisplayData(builder);
+      builder.add(DisplayData.item("tableId", tableId)
+        .withLabel("Table ID"));
```
Comment (dhalperi) [inline sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java]: Would like to see zone+cluster or instance ID as well.

Comment (dhalperi) [inline sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java]: And bigtableOptions: https://github.com/ianzhou1/incubator-beam/blob/10f8e5d8fc9ebe0c608bfd935e2b6c0131352df8/sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java#L270

**Pull request closed without merging**